ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -2810,6 +2810,86 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  interval: 4h
+  labels:
+    job-type: launch
+    job-env: openstack
+    job-release: '4.2'
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: canary-openshift-origin-installer-e2e-openstack-4.2
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-openstack-cluster-profile
+      - --target=e2e-openstack
+      - --template=/usr/local/e2e-openstack
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      command:
+      - ci-operator
+      env:
+      - name: BRANCH
+        value: "4.2"
+      - name: CLUSTER_TYPE
+        value: openstack
+      - name: CONFIG_SPEC
+        value: |
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
+          - as: e2e-$(CLUSTER_TYPE)
+            commands: TEST_SUITE=openshift/conformance/parallel run-tests
+            openshift_installer:
+              cluster_profile: "$(CLUSTER_TYPE)"
+      - name: JOB_NAME_SAFE
+        value: e2e-openstack
+      - name: KURYR_ENABLED
+        value: "false"
```

Comment:
nit: You can just drop this `KURYR_ENABLED` right?  I'd expect false to be the default.

Reply:
The default KURYR_ENABLED is currently "false" in our CI template. So yes I could drop it... but it seems that every openstack job we have in CI sets it to be explicit. Not required, but being explicit doesn't hurt and means we could easily change the default in the future without impacting the meaning of these job results.